serverless.yml

```diff
@@ -24,7 +24,7 @@ provider:
 
 functions:
   create:
-    handler: todos/create.create
+    handler: src/handler/createNote.createNote
```

Comment:
Looks nice :) . It is maybe better to name the function "handler" instead of "createNote", for example: src/handler/createNote.handler
because we already know by the name of the file what Lambda function is triggered.